### PR TITLE
cleanup gen_package.py

### PR DESCRIPTION
**Pull request by dnfield**
Won't land this TBR since it's less critical - this should do the same thing as my last PR with less code.

### CODE REVIEW

File: tools/fuchsia/gen_package.py [hunk 1]
```diff
@@ -91,20 +91,12 @@ def main():
...
 
   # Build the package
-  try:
-    output = subprocess.check_output(pm_command_base + ['build'])
-  except subprocess.CalledProcessError as e:
-    print('The "%s" command failed:' % e.cmd)
-    print(e.output)
-    raise
+  # Use check_output so if anything goes wrong we get the output.
+  output = subprocess.check_output(pm_command_base + ['build'])
```
Comment (iskakaushik) [inline tools/fuchsia/gen_package.py]: do we need to do something with this output? like logging it or does `check_output` pipe `stderr` in a way that LUCI can capture it?

Comment (dnfield) [reply tools/fuchsia/gen_package.py]: If the exception is raised, the exception object will contain the output.  It doesn't do that for `check_call`.

Actually, I think we don't even need to store the output - I was thinking that would prevent printing it but that doesn't make sense.

File: tools/fuchsia/gen_package.py [hunk 2]
```diff
@@ -91,20 +92,12 @@ def main():
...
 
   # Build the package
-  try:
-    output = subprocess.check_output(pm_command_base + ['build'])
-  except subprocess.CalledProcessError as e:
-    print('The "%s" command failed:' % e.cmd)
-    print(e.output)
-    raise
+  # Use check_output so if anything goes wrong we get the output.
+  subprocess.check_output(pm_command_base + ['build'])
```
Comment (jason-simmons) [inline tools/fuchsia/gen_package.py]: I'd like to have the script print the contents of the manifest if this command fails.

The error on LUCI is consistent with an empty `from_package` in the manifest.  However, that shouldn't be possible - `os.path.abspath` should return a nonempty string containing the current working directory path.  There might be something else wrong with the manifest.

Comment (dnfield) [reply tools/fuchsia/gen_package.py]: Added that

File: tools/fuchsia/gen_package.py [hunk 3]
```diff
@@ -90,20 +91,16 @@ def main():
...
   try:
-    output = subprocess.check_output(pm_command_base + ['archive'])
+    subprocess.check_output(pm_command_base + ['build'])
+    subprocess.check_output(pm_command_base + ['archive'])
   except subprocess.CalledProcessError as e:
-    print('The "%s" command failed:' % e.cmd)
-    print(e.output)
+    print('==================== Manifest contents =========================================')
+    with open(manifest_file, 'r') as manifest:
+      print(manifest)
```
Comment (jason-simmons) [inline tools/fuchsia/gen_package.py]: this should be `print(manifest.read())`

Comment (dnfield) [reply tools/fuchsia/gen_package.py]: Done.

**Pull request merged**